lib/harvest/user.rb

```diff
@@ -47,10 +47,12 @@ def timezone=(timezone)
       when 'mst', 'mdt' then self.timezone = 'america/denver'
       when 'pst', 'pdt' then self.timezone = 'america/los_angeles'
       else
-        if Harvest::Timezones::MAPPING[tz]
-          self["timezone"] = Harvest::Timezones::MAPPING[tz]
+        self["timezone"] = if Harvest::Timezones::MAPPING[tz]
```

Comment:
@zmoazeni was there any reason you're using `self["timezone"] = ` here instead of `self.timezone = ` as above?